src/tests/ftest/aggregation/aggregation_throttling.py

```diff
@@ -35,13 +36,14 @@ class DaosAggregationThrottling(IorTestBase):
     :avocado: recursive
     """
 
+    @skipForTicket("DAOS-")
```

Comment:
Is there going to be a ticket for this?

Reply:
Was going to skip it but now adjusted the performance until DAOS-5057 ticket is resolved. Removed this decorator.